Patch : Fixed issue 14 and 18

Rits1272 opened the pull request.

Rits1272 committed:
Patch : Fixes issue 14 and 18.

Rits1272 committed:
Patch :  Fixed issue 14 and 18.

akshatkarani commented:
Don't change the name of the file.

akshatkarani reviewed (changes requested):
This doesn't work when I only provide author name and not the category, `author` and `category` need to optional arguments.

Rits1272 commented:
Ok, I will change it.

Rits1272 committed:
Patches

Updated the function so that if the category is not given then the function returns all the quotes of the person specified,

Rits1272 committed:
Delete updated_brainyquote.py

Rits1272 committed:
Delete brainyquote.py

Rits1272 committed:
Patches

Updated the function so that if the category is not specified then the function returns all the quotes of the input author.

Rits1272 committed:
Fixing pep8 issues.

akshatkarani commented:
ping me when it's ready

Rits1272 commented:
It's ready

akshatkarani reviewed (changes requested):
Even the author parameter should be optional, if I call `get_quote()` then also it should work.

Rits1272 commented:
Ok I will also make it optional.

On Mon, 15 Jul 2019, 3:18 pm Akshat Karani, <notifications@github.com>
wrote:

> *@akshatkarani* requested changes on this pull request.
>
> Even the author parameter should be optional, if I call get_quote() then
> also it should work.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/oss2019/pyquotes/pull/28?email_source=notifications&email_token=AKQXHQR7GGNEZGC6ZKK2R63P7RBVZA5CNFSM4H7EL3BKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOB6MYXUQ#pullrequestreview-261721042>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKQXHQU2TASXUIG67V73HALP7RBVZANCNFSM4H7EL3BA>
> .
>
